CMakeLists.txt

```diff
@@ -44,21 +44,52 @@ target_compile_definitions(GSL INTERFACE
 
 # add include folders to the library and targets that consume it
 target_include_directories(GSL INTERFACE
-    $<BUILD_INTERFACE:
-        ${CMAKE_CURRENT_SOURCE_DIR}/include
-    >
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+    $<INSTALL_INTERFACE:include>
 )
 
 # add natvis file to the library so it will automatically be loaded into Visual Studio
 target_sources(GSL INTERFACE
-    ${CMAKE_CURRENT_SOURCE_DIR}/GSL.natvis
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/GSL.natvis>
 )
 
+# Check for system dir install
+set(SYSTEM_DIR_INSTALL FALSE)
```

Comment:
I don't think you need to check this, please don't.

Many people and build tool use -DCMAKE_INSTALL_PREFIX= to some custom location but expect the same layout.